src/test/mocks/archiveLoader.ts

```diff
@@ -64,7 +64,9 @@ export default () => {
     return pages && pages[pageId];
   });
 
-  const getBookIdsForPage = jest.fn((_pageId: string) => Promise.resolve([] as string[]));
+  const getBookIdsForPage = jest.fn((_pageId: string) =>
+    Promise.resolve([] as Array<{id: string, bookVersion: string}>)
```

Comment:
In `createArchiveLoader.ts` `bookVersion` can be also `undefined` so this mock probably should also reflect that